schema/tables/santa_denied.yml

```diff
@@ -8,6 +8,9 @@ examples: |-
   ```
   SELECT * FROM santa_denied;
   ```
+notes: |-
+  - Fleetd only stores the most recent 10,000 denied events to avoid hitting [osquery watchdog limits](https://fleetdm.com/guides/osquery-watchdog).
+  - "If you're running this query as a scheduled query, use the `differential_ignore_removals` to avoid hitting osquery watchdog limits"
```

Comment:
```suggestion
  - If you're running this query as a scheduled query, use the `differential_ignore_removals` to avoid hitting osquery watchdog limits
```